elasticdl/pkg/common/embedding_table.go

```diff
@@ -27,6 +27,7 @@ func (table *EmbeddingTable) GetEmbeddingVector(index int64) *Tensor {
 	}
 	// TODO(qijun) only support zero initializer now
 	newVector := NewVector(table.Dim)
+	newVector.Name = table.Name
```

Comment:
I see there are multiple assignments to the Name property of vector. How about

```go
newVector := NewVector(table.Dim, table.Name)
```

Reply:
Done